### ISSUE TITLE
json to pogo misses some json element in POJO

### ISSUE THREAD

**Issue by DMVarunRauthan** [joelittlejohn/jsonschema2pojo#592]
Hi,

I was using json2pojo to convert json to pojo; but saw a anomaly while using it at one case.
so if i have json like : 
(below files extn changes from json to TXT to upload)
[DQValidationCheck.txt](https://github.com/joelittlejohn/jsonschema2pojo/files/363709/DQValidationCheck.txt)

I am able to create three pojo files which are absolutely correct as per the json.
(below files extn changes from java to TXT to upload)
[Column.txt](https://github.com/joelittlejohn/jsonschema2pojo/files/363715/Column.txt)
[DQValidationCheck.txt](https://github.com/joelittlejohn/jsonschema2pojo/files/363716/DQValidationCheck.txt)
[Validation.txt](https://github.com/joelittlejohn/jsonschema2pojo/files/363717/Validation.txt)

however if i use the below json:
(below files extn changes from json to TXT to upload)
[DQValidationCheck.txt](https://github.com/joelittlejohn/jsonschema2pojo/files/363724/DQValidationCheck.txt)

i get 
(below files extn changes from java to TXT to upload)
[DQValidationCheck.txt](https://github.com/joelittlejohn/jsonschema2pojo/files/363733/DQValidationCheck.txt)
[Validation.txt](https://github.com/joelittlejohn/jsonschema2pojo/files/363732/Validation.txt)
[Column.txt](https://github.com/joelittlejohn/jsonschema2pojo/files/363734/Column.txt)

Check the Validation java class where the Default member is missing the second time.

Basically i did some trials and found that only the last json list item is looked upon to make POJO.
I am not sure if it supposed to be, kindly suggest.

Many Thanks,
Varun

**Comment by DMVarunRauthan** [comment]
I am only using the below config in pom.

<configuration>
		<annotationStyle>gson</annotationStyle>
		<sourceType>json</sourceType>
		<sourceDirectory>${basedir}/src/main/resources</sourceDirectory>
		<outputDirectory>${basedir}/src/main/java</outputDirectory>
</configuration>

**Comment by joelittlejohn** [comment]
Indeed, this is a limitation of the JSON handling. Since we don't support either union types or tuples, only the first element in an array is read.

**Comment by DMVarunRauthan** [comment]
ahh thanks.
